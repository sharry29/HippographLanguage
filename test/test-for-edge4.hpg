int main() {
  graph<int:int, int> g = [1:1-(3)>2:2<(5)>4:4; 7:7<(8)-6:6];
  for_edge(src, dst, w : g) {
    print_int(src.get_data());
    print_int(dst.get_data());
    print_int(w);
  }
  return 0;
}