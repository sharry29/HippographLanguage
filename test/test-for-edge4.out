1
2
3
2
4
5
4
2
5
6
7
8